content-src/asrouter/components/SnippetBase/SnippetBase.jsx

```diff
@@ -19,7 +19,12 @@ export class SnippetBase extends React.PureComponent {
       return (
         <div className="footer">
           <div className="footer-content">
-            <button className="ASRouterButton secondary" title={this.props.content.block_button_text} onClick={this.props.onDismiss}>{this.props.content.dismiss_button_label}</button>
+            <button
+              className="ASRouterButton secondary"
+              title={this.props.content.block_button_text}
```

Comment:
We should add `block_button_text` to the dummy provider.

Reply:
:+1: